test/extended/setup.sh

```diff
@@ -140,7 +140,7 @@ function os::test::extended::setup () {
 	if [[ -z "${SKIP_NODE:-}" ]]; then
 		oc rollout status dc/docker-registry
 	fi
-	DROP_SYN_DURING_RESTART=true CREATE_ROUTER_CERT=true os::start::router
+    CREATE_ROUTER_CERT=true os::start::router
```

Comment:
nit: indent the line?

Reply:
Bah... my editor hid that and replaced the tab with 4 spaces :-(  Thanks for catching it.